xml/vm_security.xml

```diff
@@ -77,7 +77,7 @@ The confidential compute module provides replacement packages supporting AMD SEV
 <step>
 <para>To install the replacement packages, run the command:</para>
 <screen>
-&prompt.sudo; <command>zypper install --from SLE-Module-Confidential-Computing-15-SP6-Pool --from SLE-Module-Confidential-Computing-15-SP6-Updates qemu libvirt</command></screen>
+&prompt.sudo; <command>zypper install --from SLE-Module-Confidential-Computing-15-SP6-Pool --from SLE-Module-Confidential-Computing-15-SP6-Updates qemu libvirt kernel-coco</command></screen>
```

Comment:
And there is still one miussing: qemu-ovmf-x86_64 
Sorry to discover this one so late.